### PR TITLE
Review and update use case documentation

### PR DESCRIPTION
**Pull request by KennethTeo2002**
Made changes to documentation based on tutorial 7 teachings.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -383,16 +370,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-2.  OS returns save file object.
-3.  For each line of save file.
+1.  Actor retrieve save file object.
+2.  For each line of save file.
 
-    - 3.1 Actor reads a line from the file.
-    - 3.2 Actor parses the line into a contact object.
-    - 3.3 Actor adds the contact object to the contact list.
-    - 3.4 Repeat steps 3.1 - 3.3 until all lines are processed.
+    - 2.1 Actor reads a line from the file.
```
Comment (ITLimJiaWei) [inline docs/DeveloperGuide.md:376]: Should this be System instead of Actor? since we define Actor as User

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -421,15 +406,13 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 **MSS**
 
-1.  Actor searches OS directory for save file.
-2.  OS returns save file object.
-3.  For each contact object.
+1.  Actor retrieve save file.
```
Comment (ITLimJiaWei) [inline docs/DeveloperGuide.md:409]: Sane as above

**Review by ITLimJiaWei — CHANGES_REQUESTED** [review]
Not sure if we should have UCs for the system? But if we do then we should specify actor in this case is the system/BizBook

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (KennethTeo2002) [reply docs/DeveloperGuide.md]: I reread the definition on Actors and it states "...Actors are not part of the system; they reside outside the system.", and thus removed the UCs.

**Review by ITLimJiaWei — APPROVED** [review]
LGTM

**Review by sheenkerr — APPROVED** [review]
LGTM

**Pull request merged**